### DISCUSSION TITLE
Regulons differed based on cell clusters

### DISCUSSION THREAD

**Discussion by bioinflix** [aertslab/SCENIC#329] [* General SCENIC questions]
Hi, I want to infer cell type specific regulons based on my scRNA-seq data. I have already finished cell type annotation, for example, cell clusters: A, B, C, D, E. When I input these cell clusters into SCENIC, it successfully inferred regulons.

However, if I only chose cell clusters A, B, and C to calculate RSS score, the cell type specific regulons were different from that of the analysis using all above cell clusters.

By the way, I tried to re-classify the cell cluster E into E1 and E2, and then input cell clusters A, B, C, D, E1, and E2, into SCENIC. The regulons for A, B, C, and D in this analysis were also different from their regulons results based on analysis using A, B, C, D, E.

So, will SCENIC infer regulons for one certain cell type also considering other cell clusters?

**Comment by lennycheung** [comment]
I got the same thing! I ran a SCENIC on a subset of clusters, and cluster X has high activation of genes A, B, and C. Genes A and B are known TFs of cluster X, and our candidate factor C was novel so that was positive news. Someone asked for us to try it on the whole dataset instead of the subset, but now I only got gene A and B! The data from the subset is of course in the full set, and expression of gene C is high in cluster X in both projections.

Any help in understanding why the SCENIC output would be different would be greatly appreciated!
